resources/js/Components/SearchForm.svelte

```diff
@@ -0,0 +1,37 @@
+<script>
+    import { useForm } from '@inertiajs/svelte';
+    import Lens from '@components/icons/Lens';
+    let form = useForm({
```

Comment:
Do we need to do this? I think this can just be a vanilla html form.

Reply:
Inertia docs said they don't recommend using vanilla html forms since they cause a full page reload. So they have a the workflow where you intercept the submission and make the request using inertia instead. The useForm is just some stuff to reduce the boilerplate you'd need for forms.